shell/mixins/resource-fetch.js

```diff
@@ -131,6 +131,10 @@ export default {
 
       const schema = this.$store.getters[`${ currStore }/schemaFor`](type);
 
+      if (!schema) {
```

Comment:
Is this needed? Can't we check for the schema before we try and find? Slight concern since this function is used extensively.